tests/Model/ModelManagerTest.php

```diff
@@ -149,6 +149,7 @@ public function testSupportsQuery(bool $expected, object $object): void
     public function supportsQueryDataProvider(): iterable
     {
         yield [true, new ProxyQuery($this->createMock(QueryBuilder::class))];
+        yield [true, $this->createMock(Query::class)];
```

Comment:
You forgot `use Doctrine\ORM\Query;`

Reply:
So I did! I need to get comfortable doing this via IDE instead of the github website, so I get warnings - added!

Reply:
Also with an IDE you can install the vendor and run the tests, seems like it's failing